benchmarks/src/main/java/org/questdb/StrPosBenchmark.java

```diff
@@ -70,10 +70,8 @@ public StrPosBenchmark() {
         strings = new String[N];
         for (int i = 0; i < N; i++) {
             builder.setLength(0);
-            int startLen = rnd.nextInt(1000);
-            for (int j = 0; j < startLen; j++) {
-                builder.append('a');
-            }
+            int startLen = rnd.nextInt(32);
+            builder.append("a".repeat(startLen));
```

Comment:
This is JDK11 only API

I specifically removed it here:

https://github.com/questdb/questdb/pull/1533

There is an open issue regarding jdk8 compatibility 

https://github.com/questdb/questdb/issues/1532

Reply:
My bad. Let me revert this part of the change.

Reply:
Fixed in 9e96fc609fad87981548f79ca193e7d9294f2bbe